.github/workflows/update-core-deps.yml

```diff
@@ -12,6 +12,17 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
+        with:
+          submodules: true  # Ensure submodules are checked out
+
+      - name: Update submodules
+        run: |
+          git submodule update --init
+          cd inputfiles/mdn  # Replace with actual submodule path
+          git checkout main  # Change if using a different branch
```

Comment:
We are certainly using a different branch (well, no branch actually), so no "if". And `git switch` is preferred for branches nowadays.